tasker/job_template/views.py

```diff
@@ -1,21 +1,16 @@
 from flask import Blueprint, render_template, redirect, url_for, flash
 from tasker.models import db
 #from tasker.job_template.forms import JobTemplateForm
+from tasker.models import JobTemplate
 
 bp = Blueprint('job_template', __name__, static_folder='../static')
 
 @bp.route('/templates')
 #@login_required
 def templates():
-    user = {
-    'username': 'test@testing.com', 'email' : 'test@testing.com', 'timezone' : 'EST', 'view' : 'Month'
-    }
-    templates = [
-    {'id': 0, 'title':'Template 1', 'start_date' : '09.10.2020', 'interval': 3, 'interval_type' : 'Weeks', 'description' : 'Job Template description goes here.'},
-    {'id': 1, 'title': 'Template 2', 'start_date' : '09.15.2020', 'interval': 5, 'interval_type' : 'Days', 'description' : 'Job Template description goes here.'},
-    {'id' : 2, 'title':'Template 3', 'start_date' : '09.20.2020', 'interval' : 1, 'interval_type' : 'Months', 'description' : 'Job Template description goes here.'}
-    ]
-    return render_template('job-template/templates.html', title='Templates', user=user, templates=templates)
+    jobs = JobTemplate.getJobTemplates()
```

Comment:
I think it would be simpler to do the query here than delegate to the JobTemplate class.  Also, the query needs to select only templates for the logged in user.

Reply:
I agree with simpler, but just trying to keep the MVC paradigm.  I meant to add the user filter, but then it slipped my mind.  I'll add that shortly.

Reply:
In Flask applications, the route (which we have been putting in a file called views, which may not be particularly accurate) acts as the controller and the template is the view. So performing the query in the route definition still upholds the MVC paradigm.